content/blog/2023/12/camel-data-types/index.md

```diff
@@ -0,0 +1,490 @@
+---
+title: "Camel 4 Data Types"
+date: 2023-12-04
+draft: false
+authors: [christophd]
+categories: ["Howtos", "Transformation"]
+preview: "Boosting the interoperability of Camel routes and Kamelets with data type transformations"
+---
+
+Since Camel 4 users are able to apply data types to their individual Camel routes in order to  transform the processed message content in a declarative way. 
```

Comment:
```suggestion
Since Camel 4, users are able to apply data types to their individual Camel routes in order to transform the processed message content in a declarative way. 
```